spec/kitchen/driver/aws/image_selection_spec.rb

```diff
@@ -32,7 +32,8 @@ def new_instance(platform_name: "blarghle")
       :provisioner => Kitchen::Provisioner::Dummy.new,
       :transport => Kitchen::Transport::Dummy.new,
       :verifier => Kitchen::Verifier::Dummy.new,
-      :state_file => Kitchen::StateFile.new("/nonexistent", "suite-name-#{platform_name}")
+      :state_file => Kitchen::StateFile.new("/nonexistent", "suite-name-#{platform_name}"),
+      lifecycle_hooks: Kitchen::LifecycleHooks.new({})
```

Comment:
Can we make these consistent `:key =>` or `key:` after we verify everything passes?